### DISCUSSION TITLE
set next.config.js output: 'standalone' then, Error: Cannot find module 'next'

### DISCUSSION THREAD

**Discussion by dmdgpdi** [vercel/next.js#72249] [Help] [answered]
### Summary

"After adding output: 'standalone' to next.config.js, I get a Cannot find module 'next' error when running node server.js in the standalone folder after next build. I'm using Yarn Berry and PnP, but I'm not using zero install. Does anyone know why this is happening?"

<img width="259" alt="스크린샷 2024-11-04 오후 12 30 38" src="https://github.com/user-attachments/assets/1a69d373-9292-4ecf-89d8-eea53fd5474d">

```js
// next.config.js
const { createVanillaExtractPlugin } = require('@vanilla-extract/next-plugin');
const withVanillaExtract = createVanillaExtractPlugin();

/** @type {import('next').NextConfig} */
const nextConfig = {
  images: {
    remotePatterns: [{ protocol: 'https', hostname: '**' }],
  },
  output: 'standalone',
};
module.exports = withVanillaExtract(nextConfig);
```

```yml
# .yarnrc.yml
nodeLinker: pnp
yarnPath: .yarn/releases/yarn-4.3.1.cjs
```

### Additional information

```js
next version: Next.js v14.2.9
yarn version: 4.3.1
```

**Comment by dmdgpdi** [comment, marked as answer]
[https://stackoverflow.com/questions/78681836/error-cannot-find-module-next-in-docker-next-14-node-20-react-18](https://stackoverflow.com/questions/78681836/error-cannot-find-module-next-in-docker-next-14-node-20-react-18) 
found answer.

**Comment by jpainam** [comment]
This is still happening with `pnpm` and `outputFileTracingRoot` has been removed. Any pointer ?

**Reply by atanevski** [reply]
I have same issue
